src/lib/actions/analytics.ts

```diff
@@ -1,18 +1,28 @@
 import { page } from '$app/stores';
 import { user } from '$lib/stores/user';
-import { ENV, MODE, VARS } from '$lib/system';
+import { ENV, MODE, VARS, isCloud } from '$lib/system';
 import googleAnalytics from '@analytics/google-analytics';
+import googleTagManager from '@analytics/google-tag-manager';
 import { AppwriteException } from '@appwrite.io/console';
 import Analytics from 'analytics';
 import { get } from 'svelte/store';
 
 const analytics = Analytics({
     app: 'appwrite',
-    plugins: [
-        googleAnalytics({
-            measurementIds: [VARS.GOOGLE_ANALYTICS || 'G-R4YJ9JN8L4']
-        })
-    ]
+    plugins: isCloud
+        ? [
+              googleAnalytics({
+                  measurementIds: [VARS.GOOGLE_ANALYTICS || 'G-R4YJ9JN8L4']
+              }),
+              googleTagManager({
+                  containerId: [VARS.GOOGLE_TAG || 'GTM-P3T9TBV']
+              })
+          ]
+        : [
+              googleAnalytics({
+                  measurementIds: [VARS.GOOGLE_ANALYTICS || 'G-R4YJ9JN8L4']
+              })
+          ]
```

Comment:
```suggestion
    plugins: [
        googleAnalytics({
            measurementIds: [VARS.GOOGLE_ANALYTICS || 'G-R4YJ9JN8L4']
        }),
        ...(isCloud
            ? [
                googleTagManager({
                    containerId: [VARS.GOOGLE_TAG || 'GTM-P3T9TBV']
                })]
            : [])
    ]
```

I think something like this might be cleaner? 🤔